Jenkinsfile

```diff
@@ -109,25 +109,36 @@ try {
   }
 }
 
+def String startSyncTestingServer() {
+  sh ''' adb reverse tcp:7800 tcp:7800 &&
+      adb reverse tcp:8080 tcp:8080 &&
+      adb reverse tcp:8888 tcp:8888 &&
+      ros-testing-server.js "sync-server.build.log" > /dev/null &
+      echo $! > sync_pid
+  '''
+  return readFile("sync_pid").trim()
```

Comment:
we can now use the `returnStdout` argument of `sh`. No need to use a file anymore: https://jenkins.io/doc/pipeline/steps/workflow-durable-task-step/#sh-shell-script